src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceBeforeCheck.java

```diff
@@ -133,6 +135,28 @@
  *        .listIterator()
  *        .forEachRemaining(System.out::print); // OK
  * </pre>
+ * <p>
+ *     To Configure the check to restrict the use of whitespace before
+ *     LABELED_STAT tokens:
+ * </p>
+ * <pre>
+ * &lt;module name=&quot;NoWhitespaceBefore&quot;&gt;
+ *   &lt;property name=&quot;tokens&quot; value=&quot;LABELED_STAT&quot;/&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * class Foo {
+ *   {
+ *     label1 : // violation, whitespace before ':' is not allowed
+ *     System.out.println();
+ *   }
+ *   {
+ *     label2: // OK
+ *     System.out.println();
```

Comment:
I know that labels can be before any statement, but for example it is better to show it with loops.

Reply:
Done.